app/assets/v2/js/shared.js

```diff
@@ -1010,6 +1010,75 @@ function shuffleArray(array) {
   return array;
 }
 
+
+const getAllUrlParams = () => {
```

Comment:
maybe this function can be a lot more simple with https://developer.mozilla.org/pt-BR/docs/Web/API/URLSearchParams